lib/ezutils/classes/eztimestamp.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * File containing the eZTimestamp class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ * @package lib
+ */
+
+class eZTimestamp
+{
+    /*!
+     \return a timestamp in UTC
+    */
+    public static function getUtcTimestampFromLocalTimestamp($localTimestamp) {
```

Comment:
Strictly speaking, this file needs space between parentheses everywhere for legacy CS: `foo( $bar )`

Reply:
```
        ____________________
  ___  |        FRESH       |
 /_| | |       SPACES       |
|    |_|__________________  |
"-O----O-O' `         `O`O'-'
```